aws-api-appsync/src/main/java/com/amplifyframework/api/aws/SelectionSet.java

```diff
@@ -276,16 +276,20 @@ private SelectionSet wrapPagination(SelectionSet node) {
          */
         private static boolean isCustomType(@NonNull Field field) {
             Class<?> cls = getClassForField(field);
-            if (Model.class.isAssignableFrom(cls) || Enum.class.isAssignableFrom(cls)) {
+            if (cls.getClassLoader() == null || (cls.getClassLoader() != null && isDateType(cls))) {
```

Comment:
Isn't including the cls.getClassLoader() != null check redundant since the cls.getClassLoader() == null check would have already resolved if true?

Reply:
@TrekSoft ah thats right, let me revise!